jenkins/debian/Jenkinsfile_basic

```diff
@@ -0,0 +1,17 @@
+library identifier: 'jenkins-libs@master', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: 'https://github.com/ana/kernelci-jenkins-libs'])
+
+buildTest {
+
```

Comment:
Pipelines should probably best described as [scripted pipelines](https://jenkins.io/doc/book/pipeline/#scripted-pipeline-fundamentals)

Reply:
Hi! These are jenkins scripted pipelines, take a look a this example: https://github.com/jenkinsci/pipeline-examples/tree/master/global-library-examples/global-function